nautobot/ipam/filters.py

```diff
@@ -302,7 +302,7 @@ class PrefixFilterSet(
         queryset=VLAN.objects.all(),
         label="VLAN (ID)",
     )
-    vlan_vid = django_filters.NumberFilter(
+    vlan_vid = django_filters.CharFilter(
```

Comment:
I think this change is incorrect - VLAN VID (as opposed to `vlan_id`) *is* a number.

Reply:
Hmm, okay good catch. Thanks. Admittedly the switch to UUID is kind of non-intuitive since switching to chars for pk/id. 🙈